selfdrive/car/gm/interface.py

```diff
@@ -48,6 +49,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), has_relay=False,
       ret.steerRatio = 15.7
       ret.steerRatioRear = 0.
       ret.centerToFront = ret.wheelbase * 0.4  # wild guess
+      ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0., 15.0], [0., 15.0]]
+      ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.2, 0.4], [0.02, 0.08]]
```

Comment:
I tried these PID tuning values in 076 and it was arguably worse than the earlier values. It was jerking the wheel back and forth more during sharper curves, at lower speeds. Does the tuning need to be on 08 to work properly?

Reply:
Reverted the tuning as it was worse at some speeds